api/src/pcapi/core/offerers/api.py

```diff
@@ -1556,7 +1556,7 @@ def _get_query(offer_class: type[offers_api.AnyOffer]) -> BaseQuery:
         return sa.select(sa.func.jsonb_object_agg(sa.text("status"), sa.text("number"))).select_from(
             sa.select(
                 sa.case(
-                    (sa.and_(offer_class.isActive.is_(True), offer_class.is_expired.is_(False)), "active"),  # type: ignore [attr-defined]
+                    (sa.and_(offer_class.isActive, sa.not_(offer_class.is_expired)), "active"),  # type: ignore [type-var]
```

Comment:
est-ce que tu sais pourquoi il faut un ignore sur le type-var ?

Reply:
Je n'ai pas creusé suffisamment pour trouver une solution, mais c'est au niveau du `and_` ou du `or_` que mypy n'est pas content (et plus de la propriété elle-même) : 
`src/pcapi/core/offerers/api.py:1559:22: error: Value of type variable "_CLE" of function cannot be "object"  [type-var]`
J'ai dû l'ajouter pour les cas qui étaient dans des `or_`, que tu verras par ailleurs, par contre ça passe sans souci dans `filter` directement (sans `and_` explicite).

Ce sera la prochaine étape, on avance pas à pas.